index.js

```diff
@@ -0,0 +1,94 @@
+import myBookShowList from './modules/BookShow.js';
+import myShowContactView from './modules/ContactShow.js';
+import myAddBookList from './modules/BookAdd.js';
+import myRedirectButton from './modules/RedirectButton.js';
+import { DateTime } from './modules/luxon.js';
+
+const Error = document.getElementById('error');
+const myText = document.getElementById('text');
+const myDate = document.getElementById('date');
+const myLuxon = DateTime.local(new Date()).c;
+console.log(myLuxon);
+myDate.innerHTML = `${myLuxon.year}, ${myLuxon.month}, ${myLuxon.day}, ${myLuxon.hour}, ${myLuxon.minute}, ${myLuxon.second}`;
+
+class BookCollection {
+  constructor() {
+    this.BOOKS_KEY = 'awesome-books';
+    this.books = JSON.parse(localStorage.getItem(this.BOOKS_KEY)) || [];
+    this.myTitle = document.getElementById('input');
+    this.myAuthor = document.getElementById('input1');
+    this.myAdd = document.getElementById('bttn');
+    this.myList = document.getElementById('list-holder');
+  }
+
+  saveBooks() {
+    localStorage.setItem(this.BOOKS_KEY, JSON.stringify(this.books));
+  }
+
+  addBook() {
+    const title = this.myTitle.value.trim();
+    const author = this.myAuthor.value.trim();
+
+    if (title && author) {
+      const book = { title, author };
+      this.books.push(book);
+      this.saveBooks();
+      this.renderBooks();
+      this.myTitle.value = '';
+      this.myAuthor.value = '';
+      Error.textContent = '';
+    } else {
+      Error.textContent = 'Please enter both title and author';
+    }
```

Comment:
- [ ] Some of your functions do not use the arrow function syntax. Please use ES6 syntax for all your functions as this is a project requirement. 🙏🏾